src/shared/components/app/footer.tsx

```diff
@@ -58,7 +59,9 @@ export class Footer extends Component<FooterProps, any> {
             </li>
             <li className="nav-item">
               <a className="nav-link" href={joinLemmyUrl}>
-                {I18NextService.i18n.t("join_lemmy")}
+                {myAuth()
+                  ? new URL(joinLemmyUrl).hostname
```

Comment:
Why these two different representations? This seems pointless.

Reply:
I'd rather it just be consistent. And if its useful to always show it, (I think it is), then that issue should just be closed.

Reply:
> Why these two different representations? This seems pointless.

After a user has logged in, the phrase, `Join Lemmy` feels out of place since they've _joined_ Lemmy (even though we're really trying to convey the join lemmy website).

I thought the URL representation conveyed "visit this website" more than the phrase `Join Lemmy`.

If it's not that important, I'm okay to just drop this PR and close the issue!

Reply:
I'd also be fine with your host version, join-lemmy.org being on both.